soh/include/z64audio.h

```diff
@@ -148,7 +148,7 @@ typedef struct
     /* 0x08 */ AdpcmLoop* loop;
     /* 0x0C */ AdpcmBook* book;
     u32 sampleRateMagicValue; // For wav samples only...
-    s32 sampleRate;           // For wav samples only...
+    float sampleRate;           // For wav samples only...
```

Comment:
seeing if i can kick off a build by committing a suggestion

```suggestion
    float sampleRate;         // For wav samples only...
```